includes/settings/js/src/components/ContentModelDropdown.jsx

```diff
@@ -58,6 +58,13 @@ export const ContentModelDropdown = ({ model }) => {
 				<Icon type="options" />
 			</button>
 			<div className={`dropdown-content ${dropdownOpen ? "" : "hidden"}`}>
+				<a
+					className="new"
+					href={`/wp-admin/post-new.php?post_type=${slug}`}
+					onBlur={() => maybeCloseDropdown(setDropdownOpen, timer)}
+				>
+					{__("New Entry", "atlas-content-modeler")}
```

Comment:
All the other items in the dropdown are verbs and/or start with verbs.

```suggestion
					{__("Add New Entry", "atlas-content-modeler")}
```